src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java

```diff
@@ -17,8 +16,12 @@ public NameContainsKeywordsPredicate(List<String> keywords) {
 
     @Override
     public boolean test(Person person) {
-        return keywords.stream()
-                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(person.getName().fullName, keyword));
+        for (String keyword: keywords) {
+            if (person.getName().fullName.toLowerCase().contains(keyword.toLowerCase())) {
+                return true;
+            }
+        }
```

Comment:
Could probably be optimised to:

```suggestion
        String lowerCaseName = person.getName().fullName.toLowerCase();
        for (String keyword: keywords) {
            if (lowerCaseName.contains(keyword.toLowerCase())) {
                return true;
            }
        }
```